### ISSUE TITLE
Exam project description - formal requirements

### ISSUE THREAD

**Issue by tnv875** [abjer/sds2019#29]
We are in doubt as to what the exam project description has to consist of and what the formal requirements are, will a description be uploaded?

**Comment by abjer** [comment]
Did you already see the formal requirements [here](https://abjer.github.io/sds2019/page/practical/)?

**Comment by tnv875** [comment]
Yes we did 👍 We were wondering about the exam project description that is due Sunday. Should this also contain a description of the data process, how long do you expect it to be and such 👍
